docs/config.md

```diff
@@ -0,0 +1,104 @@
+# Configuring the HYPR Authenticator
+To use the HYPR authenticator with WSO2 Identity Server, first you need to configure  the authenticator with
+WSO2 Identity Server. The following steps provide instructions on how to configure the HYPR authenticator with
+WSO2 Identity Server using a sample app.
+
+In order to test the approach, First an end user account should be registered in the Application created via the 
```

Comment:
Shouldn't this section be part of the `Requirements` section?
We usually call this `Prerequisites`

Reply:
Resolves via [3ef7a2c](https://github.com/wso2-extensions/identity-outbound-auth-hypr/pull/2/commits/3ef7a2caa1aafeb0302cc6662760cbe471bd299d).